apps/andi/lib/andi/secret_service.ex

```diff
@@ -29,10 +29,6 @@ defmodule Andi.SecretService do
     end
   end
 
-  def retrieve_aws_keys() do
-    retrieve("aws_keys/andi")
-  end
-
```

Comment:
Removing unused method. Only used in tests, moved to test code to make that clear.